tests/test_wave_05.py

```diff
@@ -80,34 +79,38 @@ def test_create_goal(client):
     }
 
 
-@pytest.mark.skip(reason="test to be completed by student")
+# @pytest.mark.skip(reason="test to be completed by student")
 def test_update_goal(client, one_goal):
-    raise Exception("Complete test")
     # Act
-    # ---- Complete Act Here ----
+    response = client.put("/goals/1", 
+        json={"title": "Build a habit of going outside daily"})
+    
+    response_body = response.get_json()
 
-    # Assert
-    # ---- Complete Assertions Here ----
-    # assertion 1 goes here
-    # assertion 2 goes here
-    # assertion 3 goes here
-    # ---- Complete Assertions Here ----
+    # Assert #AL
+    assert response.status_code == 200
+    assert "goal" in response_body
+    assert response_body == {
+        "goal": {
+            "title" : "Build a habit of going outside daily", 
+            "id" : 1
+        }
+    }
```

Comment:
To make your test a bit more robust, you can check that the goal was actually stored in the db by adding: 
```python
goal = Goal.query.get(1)
assert goal.title == "Build a habit of going outside daily"
    ```